package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "boilerplate",
+  "version": "1.0.0",
+  "description": "Boilerplate for node.js based micro service",
+  "main": "index.js",
+  "repository": "git@github.com:plyo/boilerplate.git",
+  "author": "Andrew Balakirev <balakirev.andrey@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "cookie-parser": "^1.4.3",
+    "express": "^4.16.3",
+    "pino": "^4.17.3",
+    "pino-elasticsearch": "^2.2.1",
+    "pino-http": "^3.2.2",
+    "superagent": "^3.8.3"
+  },
+  "devDependencies": {
+    "@plyo/dev-tools": "1.0.0-alpha.4"
+  },
+  "scripts": {
+    "lint": "lint",
+    "lint-staged": "lint --staged",
+    "precommit": "lint --staged",
+    "start": "nodemon src/server.js"
```

Comment:
I've checked nodemon for plyo.db and found out that there is no difference in performance, both solution works fast "enough". Looks like it's because approach for plyo.db should be refactored to not rerun the whole server. But for simplicity of microservices, I've added `nodemon` into `@plyo/dev-tools`

Reply:
I like proposed solution, but how would you implement it? Is it a magic way to run `start` script from `@plyo/dev-tools`?

Reply:
Example: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/bin/react-scripts.js

```js
// @plyo/dev-tools/tasks/start.js
const nodemon = require('nodemon')
nodemon({ script: process.argv[2] })
```